go/spec.go

```diff
@@ -50,6 +50,7 @@ type Field struct {
 // Service is the data parsed from TOML.
 type Service struct {
 	Name       string
+	Features   []string
```

Comment:
Should this be `Features`?

Reply:
Feature in service should be a string array like:

```toml
features = ["virtual_dir"]
```

And it's not the same type as defined in `features.toml`. We will check them while parsing `service.toml`

Reply:
I will add comments for these behavior.

Reply:
Or maybe we need to put them in namespace instead.

Reply:
PTAL, I moved features from the service global to namespace level.